dask_glm/utils.py

```diff
@@ -5,9 +5,32 @@
 
 import dask.array as da
 import numpy as np
+from functools import wraps
 from multipledispatch import dispatch
 
 
+def normalize(normalize=True):
+    def decorator(algo):
+        @wraps(algo)
+        def normalize_inputs(X, y, *args, **kwargs):
+            if normalize:
+                mean, std = da.compute(X.mean(axis=0), X.std(axis=0))
+                mean, std = mean.copy(), std.copy()
+                intercept_idx = np.where(std == 0)
```

Comment:
If `X` has multiple columns with no standard deviation, does everything still work ok? You probably have bigger problems anyway, but does this cause any issues in the new code?